<launch>

 <node pkg="progetto1" type="wheels_synchronizer" name="wheels_synchronizer" />
 <node pkg="progetto1" type="velocity_estimate" name="velocity_estimate" output="screen" />
 <node pkg="progetto1" type="scout_our_odom" name="scout_our_odom" output="screen" />
 <node pkg="tf" type="static_transform_publisher" name="world_to_odom" args="-0.832142114639 0.426361680031 0.324720174074 0.0221652016044 -0.0104456143454 -0.533567428589 -0.845402538776 world odom 100" />

<!-- Run rqt_reconfigure 
 <node type="rqt_reconfigure" name="rqt_reconfigure" pkg="rqt_reconfigure" />  -->

<!-- Run rviz 
 <node type="rviz" name="rviz" pkg="rviz" args="-d $(find progetto1)/rviz/progetto1.rviz" /> -->

 <param name="initial_pose_x" value="0.0" />
 <param name="initial_pose_y" value="0.0" />
 <param name="initial_pose_theta" value= "0.0" />

</launch>
